types/ckeditor__ckeditor5-essentials/src/essentials.d.ts

```diff
@@ -8,11 +9,9 @@ type ShiftEnter = typeof Plugin;
 type SelectAll = typeof Plugin;
 // TODO: import {Typing} from "@ckeditor/ckeditor5-typing";
 type Typing = typeof Plugin;
-// TODO: import {Undo} from "@ckeditor/ckeditor5-undo";
-type Undo = typeof Plugin;
 
 export default class Essentials extends Plugin {
-    static requires: [Clipboard, Enter, SelectAll, ShiftEnter, Typing, Undo];
+    static requires: [Clipboard, Enter, SelectAll, ShiftEnter, Typing, typeof Undo];
```

Comment:
```ts
    static requires: [Clipboard, Enter, SelectAll, ShiftEnter, Typing, typeof Undo];
```
how is this supposed to be used? the last param looks like class, not a reference

Reply:
scratch that question, realized the difference once saw the top of the file with imports